AdvancedDLSupport.Tests/AdvancedDLSupport.Tests.csproj

```diff
@@ -12,11 +12,14 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="altcover" Version="4.0.630" PrivateAssets="all" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.8.0" />
-    <PackageReference Include="xunit" Version="2.4.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
-    <PackageReference Include="JetBrains.Annotations" Version="11.1.0" PrivateAssets="all" />
+    <PackageReference Include="altcover" Version="5.0.665" PrivateAssets="all" />
```

Comment:
Whoa, what happened here? There's a bunch of updated packages, and some are major-version updates which may introduce breaking changes (altcover in particular). Out of scope for this PR.